### PR TITLE
document update

### PR DESCRIPTION
**Pull request by fansi404**
This change is [Reviewable](https://reviewable.io/reviews/koderover/zadig-doc/505)

### CODE REVIEW

File: ZadigX v1.5.0/02.运维手册/02.backup_and_restore.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ date: 2021-03-29 16:24:12
 permalink: /ZadigX v1.5.0/stable/backup-and-restore/
 ---
 
-投入生产使用后，建议定时对 ZadigX 系统的数据做备份，当数据库故障、Kubernetes 集群故障等极端情况发生时，可参考本手册快速恢复。
+投入生产使用后，建议定时对 ZadigX 系统的数据做备份，当数据库故障、Kubernetes 集群故障等极端情况发生时，可参考本手册快速恢复，对 ZadigX 进行迁移。
```
Comment (lilianzhu) [inline ZadigX v1.5.0/02.运维手册/02.backup_and_restore.md:7]: 添加迁移文档内容

File: Zadig v1.17.0/15.系统设置/17.object_storage.md [hunk 2]
```diff
@@ -17,20 +15,127 @@ permalink: /Zadig v1.17.0/settings/object-storage/
...
 > 通过配置对象存储，用户可以把构建过程产生的软件包通过工作流的分发步骤，分发到对象存储空间中。
 
-编辑带有分发步骤的工作流，可以通过勾选 `存储空间分发` 将对应服务构建产生的软件包分发到存储空间，如下图所示。
+编辑带有分发步骤的产品工作流，可以通过勾选 `存储空间分发` 将对应服务构建产生的软件包分发到存储空间，如下图所示。
 
 ![obj](./_images/obj.png)
+
+### 工作流日志
+
+工作流运行的日志会被自动上传到默认使用的对象存储中。以腾讯云 COS 示例：在指定 Bucket 下检索工作流标识 > 工作流任务序号 > log 文件夹即可。
```
Comment (lilianzhu) [inline Zadig v1.17.0/15.系统设置/17.object_storage.md:120]: 补充代码扫描、测试的日志

File: ZadigX v1.5.0/05.功能介绍/07.scan.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ permalink: /ZadigX v1.5.0/project/scan/
 - 支持在 ZadigX 中使用 SonarQube 工具和自定义工具对代码进行扫描
```
Comment (lilianzhu) [inline ZadigX v1.5.0/05.功能介绍/07.scan.md:7]: 文档开头加一句介绍

File: ZadigX v1.5.0/05.功能介绍/18.系统设置/02.代码源/00.overview.md [hunk 4]
```diff
@@ -18,19 +18,183 @@ ZadigX 支持集成以下代码源：
...
-| Webhook 触发测试 | <img style="width:100px; height:50px" src="../_images/test_webhook_config.png"></img> | ✅ | ✅ | — | ✅ |✅ | — |
-| Webhook 触发代码扫描 | <img style="width:100px; height:50px" src="../_images/scan_webhook_config.png"></img> | ✅ | ✅ | — | — | — |— |
-| 从代码库同步创建 K8s YAML 服务 |  <img style="width:100px; height:50px" src="../_images/create_k8s_service_from_repo.png"></img>| ✅ | ✅ | ✅ | ✅ |✅ | — |
-| 从代码库同步创建 K8s Helm Chart 服务 |  <img style="width:100px; height:50px" src="../_images/create_helm_service_from_repo.png"></img>| ✅ | ✅ | ✅ | ✅ |✅ | ✅ |
-| 从代码库同步创建 K8s Helm Chart 服务模板 |   <img style="width:100px; height:50px" src="../_images/create_helm_template_from_repo.png"></img>| ✅ | ✅ | ✅ | ✅ |✅ | ✅ |
-| 从代码库导入 Helm Chart values 文件 | <img style="width:100px; height:50px" src="../_images/import_values_from_repo.png"></img>| ✅ | ✅ | — |— | — | — |
-| 从代码库导入环境配置（Ingress/ConfigMap/Secret/PVC） | <img style="width:100px; height:50px" src="../_images/import_config_from_repo.png"></img>| ✅ | ✅ | — | — |— | — |
-| 工作流展示代码变更标题中的 Jira issue 信息 | <img style="width:100px; height:50px" src="../_images/show_jira_issue_info.png"></img>| ✅ | ✅ | — | — | — |— |
+不同代码源在系统中可使用的功能有所差异，详细介绍如下。
```
Comment (lilianzhu) [inline ZadigX v1.5.0/05.功能介绍/18.系统设置/02.代码源/00.overview.md:21]: 优化一下

File: ZadigX v1.5.0/02.运维手册/02.backup_and_restore.md (hunk 1 above)
Comment (fansi404) [reply ZadigX v1.5.0/02.运维手册/02.backup_and_restore.md:7]: tracked in trello issue

**Pull request merged**